io.ente.auth.yml

```diff
@@ -0,0 +1,110 @@
+app-id: io.ente.auth
```

Comment:
I don't see the relation between `https://ente.io/` and this.

Reply:
See: https://ente.io/auth/